examples/app.js

```diff
@@ -1,73 +1,96 @@
-var ISFRenderer = require('../lib/ISFRenderer')
-
-var video = null;
-var time = 0;
-function loadFile(src, callback) {
-  fetch("examples/" + src).then(function(response) {
-    response.text().then(function(body) {
-      callback(body);
-    })
-  })
+const ISFRenderer = require('../dist/build.js').Renderer;
```

Comment:
why require the built version vs the src version?

Reply:
@msfeldstein I suppose we could use babel-node?